mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping

```diff
@@ -1,10 +1,17 @@
 CLASS net/minecraft/class_1536 net/minecraft/entity/projectile/FishingBobberEntity
 	CLASS class_1537 State
 	FIELD field_7165 hookedEntity Lnet/minecraft/class_1297;
-	FIELD field_7168 luckOfTheSeaLevel I
+	FIELD field_7166 selfHitTimer I
```

Comment:
Is `self` really necessary here? Could we simply this to `hitTimer` instead?

Reply:
While `selfHitTimer > 0`, the bobber won't attach to the entity holding its fishing rod. I feel like the name `selfHitTimer` conveys that information pretty well.